public_html/layouts/basic/styles/Main.css

```diff
@@ -3211,8 +3211,8 @@ textarea.form-control {
     display: -ms-flexbox;
     display: flex;
     -webkit-box-flex: 0;
-        -ms-flex: 0 0 auto;
-            flex: 0 0 auto;
+        -ms-flex: 0 1 auto;
```

Comment:
We don't edit Main.css file directly and we avoid to edit bootstrap components. You can overwrite though bootstrap classes by adding class in tpl file which overwrite bootstrap styles (in this case you can add utility class in _Utilities.scss with prefix u- with css flex behaviour you want, to overwrite form-group class, e.g. add class 'u-flex-default' to div with 'tpl-Base-Edit-Field-MultiEmailValue').

Reply:
yes Tomasz,
it's better.